dask_match/expr.py

```diff
@@ -1003,7 +1016,7 @@ def optimize(expr: Expr, fuse: bool = True) -> Expr:
     matchpy
     optimize_blockwise_fusion
     """
-    expr, _ = simplify(expr)
+    expr = expr.simplify()
```

Comment:
Should we also add `FrameBase.simplify` so that we can avoid doing things like `df.optimize(fuse=False)` in tests/demos?